_posts/2023-07-27-trino-fest-2023-fugue-recap.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "Trino optimization with distributed caching on data lakes"
+author: "Kevin Kho and Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/Fugue.png
+show_pagenav: false
+---
+
+Fugue may be an unfamiliar name to those in the Trino ecosystem. It's another
+Python tool, a programming model built to enhance interoperability between
+Python and SQL. On the Python side of things, it's a wrapper around common tools
+like pandas and Polars which converts that code into SQL for high-performance,
+large-scale query execution. So why are we talking about it at Trino Fest?
+Because Fugue recently launched an integration with Trino, enabling that Python
+code to run on a high-powered Trino backend.
```

Comment:
thats not what it does.. it converts into SQL and that runs on Trino.  Need to reword